### ISSUE TITLE
generateFluxDiagram fails with minimal example (on local machine)

### ISSUE THREAD

**Issue by KEHANG** [ReactionMechanismGenerator/RMG-Py#479]
I used `minimal` example to generateFluxDiagram by running `python generateFluxDiagram.py testing/minimal/input.py`. It gave the error:
```
Conducting simulation of reaction system 1...
Generating flux diagram for reaction system 1...
Traceback (most recent call last):
  File "scripts/generateFluxDiagram.py", line 42, in <module>
    main()
  File "scripts/generateFluxDiagram.py", line 39, in main
    fluxdiagram.run(inputFile, useJava)
  File "/home/kehang/Code/edge_inchi_proj/RMG-Py/rmgpy/tools/fluxdiagram.py", line 517, in run
    os.path.join(rmg.outputDirectory, 'flux', '{0:d}'.format(index+1)), centralSpecies, speciesPath
  File "/home/kehang/Code/edge_inchi_proj/RMG-Py/rmgpy/tools/fluxdiagram.py", line 265, in generateFluxDiagram
    graph.write_dot(os.path.join(outputDirectory, 'flux_diagram_{0:04d}.dot'.format(frameNumber)))
  File "build/bdist.linux-x86_64/egg/pydot.py", line 1809, in <lambda>
  File "build/bdist.linux-x86_64/egg/pydot.py", line 1911, in write
  File "build/bdist.linux-x86_64/egg/pydot.py", line 2023, in create
pydot.InvocationException: Program terminated with status: 1. stderr follows: Warning: syntax ambiguity - badly delimited number '3.75015410405e' in line 8 of /tmp/tmpnMujyK splits into two tokens
Error: /tmp/tmpnMujyK: syntax error in line 8 near '-05'
```
The temporary file looks like this:
```
  1 digraph flux_diagram {
  2 label="t = 10^-6.5 s";
  3 graph [rankdir=LR, fontname=sans, fontsize=10, overlap=false, bb="0,0,788.26,243.41"];
  4 node [label="\N"];
  5 "ethane(1)" [penwidth=9.99989414208, width="0.9027", image="/home/kehang/Code/edge_inchi_proj/RMG-Py/testing/minimal/species/ethane(1).png", label=" ", pos="32.497,14    9.29", height="0.5"];
  6 "C2H5(5)" [penwidth=0.33175608147, width="0.9027", image="/home/kehang/Code/edge_inchi_proj/RMG-Py/testing/minimal/species/C2H5(5).png", label=" ", pos="133.49,149.29    ", height="0.5"];
  7 "ethane(1)" -> "C2H5(5)"  [pos="e,100.86,149.29 65.092,149.29 73.218,149.29 82.088,149.29 90.645,149.29", dir=forward, penwidth=7.28341608272];
  8 "H(6)" [penwidth=3.75015410405e-05, width="0.75", image="/home/kehang/Code/edge_inchi_proj/RMG-Py/testing/minimal/species/H(6).png", label=" ", pos="228.99,105.29", h    eight="0.5"];
  9 "ethane(1)" -> "H(6)"  [pos="e,201.84,106.89 58.003,137.78 70.599,132.31 86.346,126.14 100.99,122.29 131.06,114.39 166.14,110.05 191.81,107.74", dir=forward, penwidth    =4.43495823888];
...
```
So the issue is from the `H(6)` is having a too small penwidth or the program cannot draw such small width (look at other penwidth, they are much bigger). The penwidth setting is from the following code:
```
if concentration < concentrationTolerance:
                penwidth = 0.0
            else:
                penwidth = slope * math.log10(concentration) + maximumNodePenWidth
            node.set_penwidth(penwidth)
```
So if we increase the `concentrationTolerance`, it can solve the problem (already tested).

**Comment by nickvandewiele** [comment]
So apparently, pydot does not like numbers in scientific notation like in `penwidth=3.72013183814e-05`

**Comment by KEHANG** [comment]
yeah, I guess so from the error message of `syntax error in line 8 near '-05'`

**Comment by mliu49** [comment]
Fixed by 12c0db0d6d7f477326266f08436eb7a1b21ba780 in #480.
